lib/Screens/account_dashboard.dart

```diff
@@ -20,97 +25,453 @@ class _AccountDashboardState extends State<AccountDashboard> {
+                          fontSize: 14,
+                        ),
+                        subtitle: Text(
+                          t.to,
+                        ),
+                        trailing: TitleText(
+                          text: t.payeeName,
+                          fontSize: 14,
+                        ),
+                      ),
+                    ),
+                    Padding(
+                      padding: const EdgeInsets.fromLTRB(
+                        20.0,
+                        0,
+                        20,
+                        0,
+                      ),
+                      child: ListTile(
+                        contentPadding: const EdgeInsets.symmetric(),
+                        title: const TitleText(
+                          text: 'Date',
+                          fontSize: 14,
+                        ),
+                        // subtitle: Text(account.accountNumber),
+                        trailing: TitleText(
+                          text: t.date,
+                          fontSize: 14,
+                        ),
+                      ),
+                    ),
+                  ],
+                ),
+              ),
+            ],
+          ),
+        );
+      },
+    );
+  }
+
+  Widget _transaction(Transaction t) {
+    final icon = Icons.keyboard_arrow_right;
+    const color = Color(0xffd50000);
+
+    return ListTile(
+      onTap: () {
+        _showTransactionDetailBottomSheet(t);
+      },
+      leading: Container(
+        height: 50,
+        width: 50,
+        decoration: const BoxDecoration(
+          color: color,
+          borderRadius: BorderRadius.all(
+            Radius.circular(
+              10,
             ),
           ),
-          Padding(
-            padding: const EdgeInsets.all(
-              10.0,
+        ),
+        child: Icon(icon, color: Colors.white),
+      ),
+      contentPadding: const EdgeInsets.symmetric(),
+      title: TitleText(
+        text: t.payeeName,
+        fontSize: 14,
+      ),
+      subtitle: Text(
+        t.date,
+      ),
+      trailing: Container(
+        height: 30,
+        width: 60,
+        alignment: Alignment.center,
+        decoration: const BoxDecoration(
+          color: LightColor.lightGrey,
+          borderRadius: BorderRadius.all(
+            Radius.circular(
+              10,
             ),
-            child: RaisedButton(
-              shape: RoundedRectangleBorder(
-                borderRadius: BorderRadius.circular(
-                  18.0,
-                ),
-                side: BorderSide(
-                  color: Colors.green,
-                ),
+          ),
+        ),
+        child: Text(
+          t.amount,
+          style: GoogleFonts.muli(
+            fontSize: 12,
+            fontWeight: FontWeight.bold,
+            color: LightColor.navyBlue2,
+          ),
+        ),
+      ),
+    );
+  }
+
+  @override
+  Widget build(BuildContext context) {
```

Comment:
Can you please reorder this to be put before the private functions? In general, it would be nicer to see all the public functions before the private ones